modules/ROOT/pages/hl7/hl7-connector.adoc

```diff
@@ -17,6 +23,20 @@ release notes is listed in the See Also section of this document.
 
 You must purchase a license for the HL7 EDI Connector to use it in a production environment.
 
+== POM File Information
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-hl7-extension</artifactId>
+  <version>4.0.3</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+Mule converts RELEASE to the latest version. To specify a version, view Anypoint Exchange and click *Dependency Snippets*.
```

Comment:
Remove the first sentence. Maybe add a line of text that says to replace <version>4.0.3> with the version you are using.